models/UNet2dExample.model.yaml

```diff
@@ -5,7 +5,7 @@
 format_version: 0.3.0
 
 name: UNet2dExample
```

Comment:
Perhaps better to add a space to make it more human readable?

Reply:
I used a dash now ;)